### PR TITLE
Various fixes from latest changes

### PR DESCRIPTION
**Pull request by jameswex**
* Motivation for features / changes

Bashing bugs before witwidget 1.4 launch.

* Technical description of changes

Fixed datapoint editor panel slider to work in all environments (demos were broken) - by explicitly using the correct dom element and also ensuring some content flex-wraps.

Added checks for when extra outputs is null so it doesn't break WIT when that happens.

Allow fallback in PD plot checking when a string can't be decoded as text (for images).

Fixed boolean setting of if custom distance function set, for correct enabling/disabling of L1/L2 radio buttons.

* Detailed steps to verify changes work correctly (as executed by you)

Ran all demos and verified all features work as expected (including PD plots)
Ran notebook demos that show attributions and extra outputs, along with custom distance setting.

### CODE REVIEW

File: tensorboard/plugins/interactive_inference/utils/inference_utils.py [hunk 1]
```diff
@@ -230,7 +230,10 @@ def __init__(self, inference_address, model_name, model_type, model_version,
 
 def ensure_not_binary(value):
   """Return non-binary version of value."""
-  return value.decode() if isinstance(value, binary_type) else value
+  try:
+    return value.decode() if isinstance(value, binary_type) else value
```
Comment (stephanwlee) [inline tensorboard/plugins/interactive_inference/utils/inference_utils.py]: Can we update a test with this check?

Comment (jameswex) [reply tensorboard/plugins/interactive_inference/utils/inference_utils.py]: done

File: tensorboard/plugins/interactive_inference/utils/inference_utils.py [hunk 2]
```diff
@@ -230,7 +230,10 @@ def __init__(self, inference_address, model_name, model_type, model_version,
...
+  except UnicodeDecodeError:
```
Comment (tolga-b) [inline tensorboard/plugins/interactive_inference/utils/inference_utils.py]: when does this happen? can you put a short comment here?

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 3]
```diff
@@ -6255,7 +6255,10 @@ <h2>Show similarity to selected datapoint</h2>
             modelNum < extraOutputs.extra.length;
             modelNum++
           ) {
-            if ('attributions' in extraOutputs.extra[modelNum]) {
+            if (
+              extraOutputs.extra[modelNum] != null &&
```
Comment (tolga-b) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: I don't remember the exact code path for this right now. Can there be a case where one model returns extraOutputs and other doesn't? In that case do we still create the extraOutputs.extra array of the correct size (2) and fill one with null?

File: tensorboard/plugins/interactive_inference/utils/inference_utils.py (hunk 2 above)
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/utils/inference_utils.py]: done

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html (hunk 3 above)
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: yes that is the behavior

**Pull request merged**